src/style/footer.module.css

```diff
@@ -0,0 +1,20 @@
+.footer {
+    color: #204A6C;
+    display: flex;
+    flex-direction: column;
+    font-size: 16px;
+    font-weight: 400;
+    gap: 24px;
+}
+
+.footer p {
+    margin: 8px 0px;
+}
+
+.footer .contributorsClass .contributor {
```

Comment:
there's no real reason for this contributorsClass as far as I can see, did you need it for selector specificity? (also, nit, but having `class` in a class name is a little redundant)